source/pdo_sqlsrv/pdo_stmt.cpp

```diff
@@ -694,7 +709,7 @@ int pdo_sqlsrv_stmt_fetch( _Inout_ pdo_stmt_t *stmt, _In_ enum pdo_fetch_orienta
 
         // support for the PDO rowCount method.  Since rowCount doesn't call a method, PDO relies on us to fill the 
         // pdo_stmt_t::row_count member
-        if( driver_stmt->past_fetch_end || driver_stmt->cursor_type != SQL_CURSOR_FORWARD_ONLY ) {
+        if( driver_stmt->past_fetch_end) {
 
```

Comment:
Please add a comment why the second OR condition is no longer required (for future references when we looked back to understand the reason of removing this)